src/utils/ucc_compiler_def.h

```diff
@@ -10,11 +10,15 @@
 #include "api/ucc_status.h"
 #include <ucs/type/status.h>
 #include <ucs/sys/string.h>
+#include <ucs/debug/log_def.h>
 
 #define ucc_offsetof ucs_offsetof
 #define ucc_container_of ucs_container_of
 #define ucc_strncpy_safe ucs_strncpy_safe
 
+typedef ucs_log_component_config_t ucc_log_component_config_t;
+#define UCC_LOG_LEVEL_WARN UCS_LOG_LEVEL_WARN
```

Comment:
should we do this for all UCS log levels? If yes - shouldn't it be part of the logger header?

Reply:
moved UCC_LOG_LEVEL_WARN to the logger header. We can add other levels when it is needed (currently we only use UCC_LOG_LEVEL_WARN as default value for in global opts).